packages/shared/src/components/cards/common/CardCover.tsx

```diff
@@ -56,20 +58,13 @@ export function CardCover({
   }
 
   return (
-    <ConditionalWrapper
-      condition={shouldShowOverlay}
-      wrapper={(component) => (
-        <div className="relative flex flex-1">
-          {coverShare}
-          {component}
-        </div>
-      )}
-    >
+    <div className="relative flex flex-1">
```

Comment:
This `flex` now used all the time does not break anything else layout wise?

Reply:
Not that I can see.